eks-anywhere-common/Addons/Partner/Pulumi/pulumi-operator-helmrepository.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: pulumi-kubernetes-operator
```

Comment:
Consider naming this `pulumi` rather than `pulumi-kubernetes-operator` since, in concept, it could serve other charts too.

Reply:
I made this change.